test/classes/initializers/generics/phase1/hasLoopBad3.chpl

```diff
@@ -1,12 +1,16 @@
-// Alternate version of hasLoopBad2.chpl, where the incorrect loop actions
-// occur after we've already seen the initialization of all fields.
+// Alternate version of hasLoopBad2.chpl, where the loop occurs after all
```

Comment:
I am not concerned at all but this is maybe the kind of hybrid case we might want to
be sure we're all OK about.

Arguably this initializer isn't valid under the old definition or the new definition.
1) I recognize that we'll have some uncomfortable hybrids during the transition
2) One could claim that the new rules apply retroactively to phase 1 of "old style" initializers.

In either case this just helps us keep the pressure on to deprecate old-style initializers